integration/chart/chart.go

```diff
@@ -0,0 +1,132 @@
+// +build k8srequired
+
+package chart
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/cenkalti/backoff"
+	"github.com/giantswarm/apprclient"
+	"github.com/giantswarm/e2e-harness/pkg/framework"
+	"github.com/giantswarm/k8sportforward"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"github.com/spf13/afero"
+)
+
+type Chart struct {
+	Channel string
+	Release string
+	Tarball string
+	Name    string
+}
+
+func Push(f *framework.Host, charts []Chart) error {
```

Comment:
This got moved from `setup`  and renamed from `installInitialCharts` (because it just pushes to the cnr repo).

Reply:
Yes `chart.Push` is very meaningful and concise <3